core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -527,6 +537,19 @@ export class CkEditor4RteComponent implements AfterViewInit, OnChanges,
     });
   }
 
+  // Returns true if a rte component should not be shown in blog post editor
+  // RTE to remove it from rte configuration for the blog post editor RTE.
```

Comment:
Could you please rephrase this, it's not completely clear what this means on the first read.

Reply:
Done.